### ISSUE TITLE
### <span aria-hidden="true">❌</span> Deploy Preview for *hunger-point* failed.

### ISSUE THREAD

**Issue by rahulgundkar** [ChandrakantPrajapati0626/Food-Corner#9]
### <span aria-hidden="true">❌</span> Deploy Preview for *hunger-point* failed.

|  Name | Link |
|:-:|------------------------|
|<span aria-hidden="true">🔨</span> Latest commit | af3e931cce4b7aa837b7871124f28d8238fdcdca |
|<span aria-hidden="true">🔍</span> Latest deploy log | https://app.netlify.com/sites/hunger-point/deploys/64218747693ea0000848d7fa |

_Originally posted by @netlify[bot] in https://github.com/ChandrakantPrajapati0626/Food-Corner/issues/8#issuecomment-1485016331_
